docs/SqlBindingRelatedRepos.md

```diff
@@ -28,7 +28,9 @@ Contains templates for the various types of bindings and supported languages, ea
 
 ## Extension Bundle
 
-Currently SQL Bindings is part of the 3.x and 4.x Preview bundles.
+Currently SQL Bindings is part of the 4.x extension bundle and 3.x, 4.x preview extension bundles.
```

Comment:
Should we remove 3.x-preview here? We've only been updating 4.x-preview bundle and not 3.x-preview https://github.com/Azure/azure-functions-extension-bundles/blob/049d0636b6bd013d58c239e8de8fe04ebe58fa3d/src/Microsoft.Azure.Functions.ExtensionBundle/extensions.json#L150C5-L150C25.